resources/views/account/partials/details/wishlist.blade.php

```diff
@@ -0,0 +1,56 @@
+@props(['editable' => true])
+
+<div v-if="wishlist">
+    <div class="flex justify-between">
+        <div class="relative flex w-full justify-between mb-4 max-lg:items-end">
+            @include('rapidez-mw::account.partials.details.title')
+            @include('rapidez-mw::account.partials.details.edit-button')
+        </div>
+    </div>
+    @include('rapidez-mw::account.partials.details.description')
+    <listing v-cloak>
+        <div slot-scope="{ loaded }">
+            <reactive-base :app="config.es_prefix + '_products_' + config.store" :url="config.es_url" v-if="loaded">
+                <reactive-list id="products" component-id="products" data-field="entity_id" :default-query="function() { return { query: { terms: { 'entity_id': wishlist.items.map(e => e.product_id) } } } }">
+                    <div slot="renderResultStats"></div>
+                    <div slot="renderNoResults">@lang('This wishlist is empty.')</div>
+                    <div slot="render" slot-scope="{ data }" class="flex flex-col">
+                        <table v-if="data" class="font-sans">
+                            <x-rapidez-mw::table.header>
+                                <th class="max-md:text-start">
+                                    <p class="whitespace-nowrap">
+                                        @{{ data.length }}
+                                        <template v-if="data.length > 1">
+                                            @lang('articles')
+                                        </template>
+                                        <template v-else>
+                                            @lang('article')
+                                        </template>
```

Comment:
These are a lot of lines for an `s` 
I think a `trans`/`__` & `trans_choice` function & filter in the rapidez/core would be a good addition

So we can basically use the same as https://laravel.com/docs/10.x/localization#retrieving-translation-strings & https://laravel.com/docs/10.x/localization#pluralization

So in vue we can simply use the following examples:
Filter example:
`@{{ 'article|articles' | trans_choice(['count' => data.length]) }}`
function example:
`@{{ trans_choice( 'article|articles', ['count' => data.length]) }}`

Reply:
I see one small problem with this and that's that the Laravel translation will then not be available. We could, of course, throw this translation into the `config.translations` bucket, but I would rather avoid that as it's notoriously annoying to work with.